### PR TITLE
Add UK National Insurance number generator

### PR DESCRIPTION
**Pull request by krsriq**
This PR 

- [x] Adds a new feature ...
- [x] Covered by tests
- [ ] Fixes #nnnn

### CODE REVIEW

File: src/Faker/Provider/en_GB/Person.php [hunk 1]
```diff
@@ -90,4 +90,25 @@ class Person extends \Faker\Provider\Person
...
+    /**
+     * Generates a random National Insurance number.
+     * @link https://www.gov.uk/hmrc-internal-manuals/national-insurance-manual/nim39110
+     */
+    public function nino(): string
+    {
+        do {
+            $prefixLetters = strtoupper(static::lexify('??'));
+        } while (
+            in_array($prefixLetters[0], ['D', 'F', 'I', 'Q', 'U', 'V'])
```
Comment (IonBazan) [inline src/Faker/Provider/en_GB/Person.php:103]: how about choosing a random element from: `['?O', '?D', ?'F', ?'I', '?Q', ?'U', '?V', 'D?', 'F?', 'I?', 'Q?', 'U?', 'V?', 'BG', 'GB', 'NK', 'KN', 'TN', 'NT', 'ZZ']` instead and then using `lexify` on the result?

**Comment by bramceulemans** [comment]
Looks good to me, can you also document this in FakerPHP/fakerphp.github.io

**Comment by krsriq** [comment]
> Looks good to me, can you also document this in https://github.com/FakerPHP/fakerphp.github.io?

Yes, will do!

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
Thanks!

**Pull request merged**